File: Data Structures & Algorithms/pow-x-n/submission-2.js
class Solution {
    /**
     * @param {number} x
     * @param {number} n
     * @return {number}
     */

    helper(x, n) {
        if(n === 0)
            return 1;
        if(n === 1)
            return x;
        return x * this.helper(x, n - 1);
    }

    myPow(x, n) {
        var res = this.helper(x, Math.abs(n));

        return n >= 0 ? res : 1 / res;
    }
}

File: Data Structures & Algorithms/score-of-a-string/submission-2.cs
public class Solution {
    public int ScoreOfString(string s) {
        int res = 0;
        int j = 0;
        for(int i = 1; i < s.Length; i++) {
            int d1 = (int) s[i];
            int d2 = (int) s[j];
            res += Math.Abs(d1 - d2);
            j++;
        }
        return res;
    }
}